jdaviz/configs/default/plugins/subset_plugin/tests/test_subset_plugin.py

```diff
@@ -25,3 +36,142 @@ def test_subset_definition_with_composite_subset(cubeviz_helper, spectrum1d_cube
+                          'radius_y': {'pixel_name': 'Y Radius (pixels)', 'wcs_name':
+                                       'Dec Radius (degrees)', 'initial_value': 5.,
+                                       'final_value': 6.},
+                          'theta': {'pixel_name': 'Angle', 'wcs_name':
+                                    'Angle', 'initial_value': 0.,
+                                    'final_value': 45.}}
+circ_annulus_subset_info = {'xc': {'pixel_name': 'X Center (pixels)', 'wcs_name':
+                                   'RA Center (degrees)', 'initial_value': 5.,
+                                   'final_value': 6.},
+                            'yc': {'pixel_name': 'Y Center (pixels)', 'wcs_name':
+                                   'Dec Center (degrees)', 'initial_value': 5.,
+                                   'final_value': 6.},
+                            'inner_radius': {'pixel_name': 'Inner Radius (pixels)', 'wcs_name':
+                                             'Inner Radius (degrees)', 'initial_value': 2.,
+                                             'final_value': 3.},
+                            'outer_radius': {'pixel_name': 'Outer Radius (pixels)', 'wcs_name':
+                                             'Outer Radius (degrees)', 'initial_value': 5.,
+                                             'final_value': 6.}}
+rectangular_subset_info = {'xmin': {'pixel_name': 'Xmin (pixels)', 'wcs_name':
+                                    'RA min (degrees)', 'initial_value': 5.,
+                                    'final_value': 6.},
+                           'xmax': {'pixel_name': 'Xmax (pixels)', 'wcs_name':
+                                    'RA max (degrees)', 'initial_value': 6.,
+                                    'final_value': 7.},
+                           'ymin': {'pixel_name': 'Ymin (pixels)', 'wcs_name':
+                                    'Dec min (degrees)', 'initial_value': 2.,
+                                    'final_value': 3.},
+                           'ymax': {'pixel_name': 'Ymax (pixels)', 'wcs_name':
+                                    'Dec max (degrees)', 'initial_value': 5.,
+                                    'final_value': 6.}}
+
+
+@pytest.mark.parametrize("roi_class, subset_info", [(CircularROI, circle_subset_info),
+                                                    (EllipticalROI, elliptical_subset_info),
+                                                    (CircularAnnulusROI, circ_annulus_subset_info),
+                                                    (RectangularROI, rectangular_subset_info)])
+def test_circle_recenter_linking(roi_class, subset_info, imviz_helper, image_2d_wcs):
+
+    arr = np.ones((10, 10))
+    ndd = NDData(arr, wcs=image_2d_wcs)
+    imviz_helper.load_data(ndd, data_label='dataset1')
+    imviz_helper.load_data(ndd, data_label='dataset2')
+    # force link to be pixel initially.
+    link_image_data(imviz_helper.app, link_type='pixels')
```

Comment:
Use the helper method. Though I do not think this is necessary at all. Imviz links by pixel on load.
```suggestion
    imviz_helper.link_data(link_type='pixels')
```